client/my-sites/customer-home/style.scss

```diff
@@ -18,6 +18,17 @@
 		grid-column: $col-start / span $span;
 }
 
+@font-face {
+	font-display: swap;
+	font-family: 'Recoleta';
+	font-weight: 400;
+	src: url( 'https://s1.wp.com/i/fonts/recoleta/400.eot' );
+	src: url( 'https://s1.wp.com/i/fonts/recoleta/400.eot?#iefix' ) format( 'embedded-opentype' ),
```

Comment:
Note: `eot` is a legacy format, so it should probably be the last one on the list, since the list is ordered by preference.

I'm guessing the ordering is done this way to avoid an IE bug somewhere? In any case, `eot` appears to have only ever been supported by IE, so this should not cause other browsers to load `eot` instead of e.g. `woff2`. Testing the live branch with Chrome confirms this 👍

Reply:
i'm guessing we took the ordering from the following diff https://github.com/Automattic/wp-calypso/pull/40828/files#diff-c592a20416d21f0dbb9ed0d6bf2bb7d2R7 

@simison do you happen to know of if this was an IE workaround?

Reply:
I grabbed it from WordPress.com front marketing page if I recall correctly.

We're now adding Recoleta to a few places independently — I'm itching to find a centralized place to define importable mixins for it. 😁 

Something like this + font import:

https://github.com/Automattic/wp-calypso/blob/3d20c7437a192b45a575a2810663c47c55281603/client/landing/gutenboarding/mixins.scss#L4-L8

Additionally, I'm itching also to have a typography system available either in Gutenberg core or in Calypso, so that we could re-use it all-across. We have some makeshift mixins to [achieve it now in Gutenboarding](https://github.com/Automattic/wp-calypso/blob/3d20c7437a192b45a575a2810663c47c55281603/client/landing/gutenboarding/mixins.scss).

Our UI is a little odd in a way that it tries to be a mixture between marketing pages and Gutenberg/core-components, though.

Oookay that was a tangent, sorry! haha.

Reply:
> Additionally, I'm itching also to have a typography system available either in Gutenberg core or in Calypso

This one is used across different app bundles, so maybe as a shared package, like calypso-typography? We can experiment in follow ups

cc @Automattic/dotcom-manage-design

Reply:
> Note: eot is a legacy format, so it should probably be the last one on the list, since the list is ordered by preference.

@jaswrks @Automattic/martech does anyone recall what the IE issue was for this ordering? I think it was originally added in D22198-code Just double checking that this is intentional.

Reply:
Seems we can just stick with the WOFF and WOFF2 formats. See https://github.com/Automattic/wp-calypso/pull/41045#issuecomment-613315206